src/utils/export-map.ts

```diff
@@ -1109,6 +1109,7 @@ function childContext(
     settings,
     parserOptions,
     parserPath,
+    languageOptions,
```

Comment:
The `cacheKey` should consider `languageOptions.parser.meta` in this case?

Reply:
Most likely yes, since `parserPath` is now included in the key for legacy config. I also think, that `sourceType` and `ecmaVersion` from `languageOptions` should be added